README.md

```diff
@@ -132,3 +132,11 @@ set "JAVA_HOME=%USERPROFILE%\apps\java-1.8.0"
 set "INCLUDE=%INCLUDE%;%OPENSSL_64%\include"
 mvn clean install
 ```
+
+Contributions
+-------------
+
+Our [contribution guide](https://github.com/wildfly-security/wildfly-openssl/blob/master/CONTRIBUTING.md) will guide you 
+through the steps for getting started on the Elytron Web project and will go through how to format and submit your first PR.
```

Comment:
Small change, the `getting started on the Elytron Web project` should be replaced with `getting started on the WildFly OpenSSL project`